geode-wan/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/AlterRegionCommandWithRemoteLocator.java

```diff
@@ -0,0 +1,199 @@
+    int locator2Port = locator2.getPort();
+    Properties serverProp = new Properties();
+    serverProp.setProperty("off-heap-memory-size", "5m");
+    server1 = lsRule.startServerVM(2, server -> server.withConnectionToLocator(locator2Port)
+        .withSystemProperty("gemfire.preAllocateDisk", "false").withProperties(serverProp));
+    server2 = lsRule.startServerVM(3, server -> server.withConnectionToLocator(locator2Port)
+        .withSystemProperty("gemfire.preAllocateDisk", "false").withProperties(serverProp));
+    server3 = lsRule.startServerVM(4, server -> server.withConnectionToLocator(locator2Port)
+        .withSystemProperty("gemfire.preAllocateDisk", "false").withProperties(serverProp));
+    server4 = lsRule.startServerVM(5, server -> server.withConnectionToLocator(locator2Port)
+        .withSystemProperty("gemfire.preAllocateDisk", "false").withProperties(serverProp));
+    server5 = lsRule.startServerVM(6, server -> server.withConnectionToLocator(locator2Port)
+        .withSystemProperty("gemfire.preAllocateDisk", "false").withProperties(serverProp));
+    server6 = lsRule.startServerVM(7, server -> server.withConnectionToLocator(locator2Port)
+        .withSystemProperty("gemfire.preAllocateDisk", "false").withProperties(serverProp));
+  }
+
+  @After
+  public void cleanup() throws Exception {
+    gfsh.connectAndVerify(locator2);
+    gfsh.execute("destroy region --name=/Positions");
+    gfsh.execute("destroy region --name=/RealTimePositions");
+    gfsh.execute("destroy region --name=/Transactions");
+    gfsh.execute("destroy region --name=/Accounts");
+    gfsh.execute("destroy region --name=/RealTimeTransactions");
+    gfsh.execute("destroy region --name=/AccountBalances");
+    gfsh.execute("destroy region --name=/FxRates");
+    gfsh.execute("destroy region --name=/AssetClasses");
+    gfsh.execute("destroy region --name=/MarketPrices");
+    gfsh.execute("destroy region --name=/Currency");
+    gfsh.execute("destroy region --name=/Securities");
+    gfsh.execute("destroy region --name=/SecurityCrossReferences");
+    gfsh.execute("destroy region --name=/Visibility");
+
+    gfsh.execute("destroy gateway-sender --id=serialSender1");
+    gfsh.execute("destroy gateway-sender --id=serialSender2");
+    gfsh.execute("destroy gateway-sender --id=parallelPositions");
+    gfsh.execute("destroy gateway-sender --id=parallelTransactions");
+    gfsh.execute("destroy gateway-sender --id=parallelAccountBalances");
+    gfsh.execute("destroy gateway-sender --id=parallelRealTimePositions");
+    gfsh.execute("destroy gateway-sender --id=parallelRealTimeTransactions");
+    gfsh.execute("destroy gateway-sender --id=parallelAccounts");
+
+    gfsh.execute("destroy disk-store --name=DEFAULT");
+    gfsh.execute("destroy disk-store --name=gateway_store");
+    expectedEx.remove();
+  }
+
+  @Test
+  public void whenAlteringMultipleRegionWithAlterCommandToAddGatewaySendersThenTestShouldNotHang()
```

Comment:
Can we change the test name not to include "hang"; something like testAlteringMutipleRegionWithAlterCommand...

Reply:
sure will do it.

Reply:
there was an email chain regarding the naming of the tests in which we are moving away from test<feature> naming to when<situation>then<expected results> test naming format. I will remove the word hang